verifier/verifier.go

```diff
@@ -478,7 +502,7 @@ func executePlugin(ctx context.Context, installedPlugin plugin.Plugin, trustPoli
 		TrustPolicy:  policy,
 		PluginConfig: pluginConfig,
 	}
-
+	logger.Debugf("Plugin VerifySignatureRequest: %+v", *req)
```

Comment:
Same as #224

Reply:
Especially, we are printing binaries `req.Signature`.

Reply:
We should not print binaries to the log.

Reply:
Removed this line.